lib/core/output_sink/console_sink.rb

```diff
@@ -13,37 +13,38 @@ module Core::OutputSink
   class ConsoleSink < Core::OutputSink::BaseSink
     def ingest(document)
       print_header 'Got a single document:'
-      Utility::Logger.info document
+      puts document
```

Comment:
my gut feel is that it's better to use a logger than `puts`. My guess is that this change makes it easier to test, though?

what if you gave this class its own `print` function, so that the tests could say something like:
```
expect(subject).to receive(:print).with(includes(expected_value))
```
?

That way the output could still get the nice configs that come from a real logger (and the ability to turn the output on/off with log level, change highlighting, etc), but still easy testability?

Reply:
This is a great idea. you can still have `puts` implemented in `print`, and it's easy to change later in one place instead of search and replace in the whole file.

> my gut feel is that it's better to use a logger than `puts`. My guess is that this change makes it easier to test, though?

I guess the main reason to use `puts` instead of `logger` for console (not only this file, but also CLI), is that logger logs a message like

> INFO -- : a message

And we really want is simply `a messsage`

Reply:
There's not much overhead added when testing the logic

I can indeed change it to `print` and do `puts` inside.

Why I changed it to `puts`: this sink is not used by default, you can use it manually when needed, so when you use it you use it with intent to see the documents that would have been ingested into the Elasticsearch instance. Since there is this intent, I think it makes sense to always output the information, not based on log level.